src/components/shared/thread/ActivityGraphCanvas.js

```diff
@@ -102,12 +104,13 @@ export class ActivityGraphCanvas extends React.PureComponent<CanvasProps> {
       categories,
       enableCPUUsage,
       maxThreadCPUDelta,
+      width,
+      height,
     } = this.props;
 
-    const rect = canvas.getBoundingClientRect();
```

Comment:
I also changed some other locations like here and below to avoid calling `getBoundingClientRect`. This wasn't strictly necessary but this is a bit simpler.